README.md

```diff
@@ -9,7 +9,10 @@ Binaries in releases are build using `musl` toolchain, therefore they have no ex
 
 Add the following to your `.bashrc`
 ```
-alias kx='source <("k8s-kx")'
+kx() {
+    KX_RUN=$(/usr/local/sbin/k8s-kx)
```

Comment:
could you please remove the path? I'd prefer to leave the location of the binary up to the user.
```suggestion
    eval $(k8s-kx)
```
LGTM with this minor change